src/activemasternode.cpp

```diff
@@ -86,51 +86,6 @@ void CActiveMasternode::ManageStatus()
             return;
         }
         pnode->Release();
-
-        // Choose coins to use
-        CPubKey pubKeyCollateralAddress;
-        CKey keyCollateralAddress;
-
-        if (GetMasterNodeVin(vin, pubKeyCollateralAddress, keyCollateralAddress)) {
-            if (GetInputAge(vin) < MASTERNODE_MIN_CONFIRMATIONS) {
-                status = ACTIVE_MASTERNODE_INPUT_TOO_NEW;
```

Comment:
with this removal, this state `ACTIVE_MASTERNODE_INPUT_TOO_NEW ` will never be set.

If it's not used anywhere, we can remove it from the states list.

Reply:
Actually, I think it would be better to restore the age check (it's useful for the operator in that case).